iModelCore/BeSQLite/SQLite/blockcachevfsd.c

```diff
@@ -810,12 +810,13 @@ static int main_copy(int argc, char **argv){
   return rc;
 }
 
-static void filesCb(void *pApp, int rc, char *z){
+static int filesCb(void *pApp, int rc, char *z){
```

Comment:
```suggestion
// BENTLEY CHANGE
static int filesCb(void *pApp, int rc, char *z){
```

Reply:
We should also report this to sqlite as it there source. Updating source from cloud sqlite might accidentally override this fix again .

Reply:
https://sqlite.org/cloudsqlite/forumpost/b2679c7a61